fastly/constants.go

```diff
@@ -11,3 +11,682 @@ const TimestampFormatDescription = "The `strftime` specified timestamp formattin
 
 // SnippetTypeDescription describes the VCL snippet location.
 const SnippetTypeDescription = "The location in generated VCL where the snippet should be placed (can be one of `init`, `recv`, `hash`, `hit`, `miss`, `pass`, `fetch`, `error`, `deliver`, `log` or `none`)"
+
+// LoggingFormatUpdate is the generic logging format used for tests.
+const LoggingFormatUpdate = "%h %l %u %t \"%r\" %>s %b"
+
+// LoggingBigQueryDefaultFormat - Default format for BigQuery logging.
```

Comment:
The standard for `godoc` documentation is full sentences, so all of these should replace '- Default' with 'is the default'.